terraform/environments/nomis/application_variables.tf

```diff
@@ -104,7 +104,21 @@ locals {
           oracle_sids            = ["PCNMAUD"]
           oracle_app_disk_size = {
             "/dev/sdb" = 100  # /u01
-            "/dev/sdc" = 5120 # /u02
+            "/dev/sdc" = 2048 # /u02
+          }
+        },
+        NOMIS = {
+          always_on              = true
+          ami_name               = "nomis_db_STIG-2022-04-26*"
+          instance_type          = "r6i.4xlarge"
+          asm_data_capacity      = 4000
+          asm_flash_capacity     = 1000
+          description            = "Copy of Production NOMIS CNOM database in Azure PDPDL00035, replicating with PDPDL00035, a replacement for PDPDL10036."
+          termination_protection = true
+          oracle_sids            = ["MISPD,CNOMP"]
```

Comment:
Have you confirmed with Sandhya the oracle_sids of the DBs that go on this server? The naming scheme for dbs is different for MP and FixNGo

Reply:
Have now understood @julialawrence point properly and made the changes to the naming following her checking with Sandhya